src/ontology/config/ncit_exclusions.tsv

```diff
@@ -1,4 +1,5 @@
 term_id	term_label	exclusion_reason	exclude_children
+NCIT:C3367	Finding	MONDO:excludeNonDisease	TRUE
```

Comment:
@sabrinatoro This is all that I think needed to be added.

Reply:
@sabrinatoro I'm surprised though that after doing this, 5,229 less items are showing up in the exclusion tables, rather than more rows. Perhaps this is just because of other recent changes in NCIT.

Note though that I'm not sure how out-of-date what I have for NCIT is. Could be 2 months old? We've been running into [memory issues](https://github.com/monarch-initiative/mondo-ingest/issues/192), so we're stuck with this version of NCIT for now, and I don't know when we'll solve it.